apps/design-system/styles/mdx.css

```diff
@@ -15,7 +15,7 @@
 }
 
 [data-rehype-pretty-code-fragment] {
-  @apply relative text-white;
+  @apply relative;
```

Comment:
Fixes an issue where codeblocks starting with ```svg (and probably anything outside of ```jsx etc) showed white text on white background.

I couldn’t figure out how to use pretty code fragments for SVG specifically. One for another day.